app/views/tags/new.html.erb

```diff
@@ -30,7 +30,7 @@
         <% types.each do |type| %>
           <li>
             <%= f.radio_button("type", type) %>
-            <%= f.label("type_#{type.downcase}", type) %>
+            <%= f.label("type_#{type.downcase}", t("tag_types.#{type.underscore}").capitalize ) %>
```

Comment:
The tests are giving translation missing errors on this page, e.g. `<label for="tag_type_fandom">&lt;span class=&quot;translation_missing&quot; title=&quot;translation missing: en.tag_types.fandom&quot;&gt;fandom&lt;/span&gt;</label>`

Potentially also relevant, these are currently the singular form of the class name, e.g. Fandom instead of Fandoms.